app/src/main/kotlin/com/bitwarden/authenticator/data/authenticator/manager/TotpCodeManager.kt

```diff
@@ -14,16 +16,9 @@ interface TotpCodeManager {
     /**
      * Flow for getting a DataState with multiple verification code items.
      */
-    fun getTotpCodesStateFlow(
-        itemList: List<AuthenticatorItemEntity>,
-    ): StateFlow<DataState<List<VerificationCodeItem>>>
-
-    /**
-     * Flow for getting a DataState with a single verification code item.
-     */
-    fun getTotpCodeStateFlow(
-        item: AuthenticatorItemEntity,
-    ): StateFlow<DataState<VerificationCodeItem?>>
```

Comment:
There are no consumers observing a single code as opposed to a list, so I went ahead and removed this.